test_archive.pl

```diff
@@ -166,7 +166,13 @@
 %   SWI-Prolog tree.
 
 create_archive_file(ArchivePath, ArchiveSourceDir, FilesOut, ExampleSourceFile) :-
-    Files = [swi('include/SWI-Prolog.h'), library('archive.pl'), swi('swipl.rc')],
+    % The files have been chosen to have different first lines
+    Files = [swi('doc/packages/examples/pldoc/README'),
```

Comment:
This is problematic.   We do not want to make this package depend on others if we can avoid that.   Also the documentation (and examples) may not be installed.

Reply:
I'll revert this change and use the entire file's contents rather than the first line (which will also allow testing that large files are handled properly)